CircleciScripts/prepare_xcframework_archives.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+import shutil
+import re
+
+from framework_list import xcframeworks
+from functions import log, run_command
+
+def create_archives(xcframework_path, archive_path, version):
+    os.makedirs(archive_path, exist_ok=True)
+    for framework in xcframeworks:
+
+        finalZipName = f"{framework}-{version}"
+        log(f"Creating zip file for {finalZipName}")
+
+        xcframework_sdk_path = f"{xcframework_path}/{framework}.xcframework"
+        temp_path = f"{xcframework_path}/{framework}/{framework}.xcframework"
+
+        log(f"Copying the zip to a temp location {temp_path}")
+        shutil.copytree(xcframework_sdk_path, temp_path)  
+
+        log(f"Make the archive and move to the archives dir")
+        shutil.make_archive(finalZipName, 'zip', root_dir=xcframework_path, base_dir=f"{framework}")
```

Comment:
nits:
1. Use double-quotes
2. `framework` doesn't need to be string interpolated, right--it's already a string name?

```suggestion
        shutil.make_archive(finalZipName, "zip", root_dir=xcframework_path, base_dir=framework)
```